[2021-07-20 20:20:06,964] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: Fuse.fuse5 2021-07-19T00:06:00+00:00 [queued]>
[2021-07-20 20:20:06,982] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: Fuse.fuse5 2021-07-19T00:06:00+00:00 [queued]>
[2021-07-20 20:20:06,982] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-07-20 20:20:06,982] {taskinstance.py:1068} INFO - Starting attempt 1 of 8
[2021-07-20 20:20:06,983] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-07-20 20:20:07,065] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): fuse5> on 2021-07-19T00:06:00+00:00
[2021-07-20 20:20:07,070] {standard_task_runner.py:52} INFO - Started process 1202274 to run task
[2021-07-20 20:20:07,084] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'Fuse', 'fuse5', '2021-07-19T00:06:00+00:00', '--job-id', '335', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpfwbabmuq', '--error-file', '/tmp/tmpren4gf0u']
[2021-07-20 20:20:07,085] {standard_task_runner.py:77} INFO - Job 335: Subtask fuse5
[2021-07-20 20:20:07,241] {logging_mixin.py:104} INFO - Running <TaskInstance: Fuse.fuse5 2021-07-19T00:06:00+00:00 [running]> on host deepika-Inspiron-5570
[2021-07-20 20:20:07,440] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=sthadpka93@gmail.com
AIRFLOW_CTX_DAG_OWNER=postgres
AIRFLOW_CTX_DAG_ID=Fuse
AIRFLOW_CTX_TASK_ID=fuse5
AIRFLOW_CTX_EXECUTION_DATE=2021-07-19T00:06:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-07-19T00:06:00+00:00
[2021-07-20 20:20:10,746] {python.py:151} INFO - Done. Returned value was: None
[2021-07-20 20:20:10,749] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=Fuse, task_id=fuse5, execution_date=20210719T000600, start_date=20210720T143506, end_date=20210720T143510
[2021-07-20 20:20:11,008] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-07-20 20:20:11,023] {local_task_job.py:151} INFO - Task exited with return code 0
